dataplane/standalone/sai_entrypoint.cc

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "translator.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// TODO: implement this without using gRPC.
+sai_status_t sai_api_initialize(_In_ uint64_t flags, _In_ const sai_service_method_table_t *services)
+{
+    initialize(GoInt(50000));
```

Comment:
Also include lucius_clib.h, sai.h, and the forwarding proto library -- don't rely on transitive includes to allow translator.h to change its own includes.